modules/understanding-agent-install.adoc

```diff
@@ -59,7 +59,7 @@ Recommended cluster resources for the following topologies:
 |Compact cluster|3|0 or 1|8 vCPUs|16 GB of RAM|120 GB
 |HA cluster|3|2 and above |8 vCPUs|16 GB of RAM|120 GB
 |====
-
+// Should I also add a row to this table for 4/5 control plane clusters?
```

Comment:
I wonder if HA cluster can now be defined as 3 to 5 control plane nodes? Do you know if this has been used elsewhere?

Reply:
I did a search in the docs and I couldn't find any references like that. If you want though, I can change the number of control plane nodes on line 60 to "3 and above"

Reply:
3 to 5